sandbox-backend/src/main/java/com/exadel/sandbox/team5/dao/CompanyDAO.java

```diff
@@ -11,4 +12,14 @@
 public interface CompanyDAO extends JpaRepository<Company, Long> {
 
     List<Company> findAllByLocationsId(long id);
+
+    @Query(value = """
+            SELECT new com.exadel.sandbox.team5.util.PairSL(c.name, COUNT(o.id))
+            FROM Company c
+                LEFT JOIN Discount d ON d.company.id = c.id
+                LEFT JOIN Order o ON d.id=o.discount.id
+            WHERE c.id=d.company.id AND d.id=o.discount.id
+                GROUP BY c.name
+            """)
+    List<PairSL> getAllOrders();
```

Comment:
Please change method name to understand what it can be used for

Reply:
done